packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -564,9 +565,10 @@ describe('KeyringController', () => {
       { data: JSON.stringify(msgParams), from: account },
       SignTypedDataVersion.V4,
     );
-    const recovered = recoverTypedSignature_v4({
+    const recovered = recoverTypedSignature({
       data: msgParams as any,
-      sig: signature as string,
+      signature: signature as string,
```

Comment:
Seems like you ought to be able to take this typecasting off since `signTypedMessage` returns a string, is that the case?

Reply:
You're quite right sir!

Reply:
removed [here](https://github.com/MetaMask/controllers/pull/914/commits/e075f5bed33d0d40f63b9386a0dce773b3bc8166)